docs/content/build.md

```diff
@@ -0,0 +1,41 @@
+**Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
+
+-----
+## Building workers
+
+Before running a deployment of your game, you need to configure your [workers](https://docs.improbable.io/reference/latest/shared/glossary#worker), configure the build of your workers and also build the workers. This applies to both server-workers and client-workers.
+
+### Configuring your workers
+To create the correct configurations for each of your server-workers and client-workers, you need to use the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli) (SpatialOS documentation).
```

Comment:
This says that you need to use the CLI - but you don't need to

Reply:
will remove that part completely